app/src/main/java/cm/aptoide/pt/install/AutoUpdate.java

```diff
@@ -139,7 +139,7 @@ private void requestUpdateSelf(final AutoUpdateInfo autoUpdateInfo) {
     AptoideApplication.setAutoUpdateWasCalled(true);
 
     ContextThemeWrapper wrapper = new ContextThemeWrapper(activity,
-        activity.obtainStyledAttributes(new int[] { R.attr.alertDialog })
+        activity.obtainStyledAttributes(new int[] { R.attr.dialogTheme })
```

Comment:
There is no need for this context wrapper anymore. Since we have the activity, we can just remove the wrapper and pass the activity in the builder of the dialog.